docs/source/data-types/timeuuid.md

```diff
@@ -19,6 +19,41 @@ session
     .query("INSERT INTO keyspace.table (a) VALUES(?)", (to_insert,))
     .await?;
 
+// Read timeuuid from the table
+if let Some(rows) = session.query("SELECT a FROM keyspace.table", &[]).await?.rows {
+    for row in rows.into_typed::<(CqlTimeuuid,)>() {
+        let (timeuuid_value,): (CqlTimeuuid,) = row?;
+    }
+}
+# Ok(())
+# }
+```
+
+To use the Timeuuid on `uuid` crate, enable the feature `v1` in your crate using:
```

Comment:
You don't need to to this unless you want to create your own uuid v1 objects based on timestamp. Fetching objects from DB works perfectly fine without it.
I think this should be more like "To create your own `Timeuuid` objects from timstamp-base `uuid` v1, you need to enable the feature `v1` of `uuid` crate using:"